server/src/db/serializers/UserSerializer.js

```diff
@@ -0,0 +1,7 @@
+class UserSerializer {
+    static showUsername (user) {
```

Comment:
nit picking - no space before the parameter parenthesis
https://github.com/airbnb/javascript#whitespace--around-keywords